### PR TITLE
Added Any.do

### LINKED ISSUE #2114: Add Any.do
```
name: Any.do
url: https://web.any.do/
img: anydo.png
tfa: No
twitter: AnyDO
facebook: Any.DO
```

image

### PR DESCRIPTION
**Pull request by moritzsternemann**
Fixed #2114

### CODE REVIEW

**Review by stephengroat — APPROVED** [review]

**Comment by stephengroat** [comment]
thanks for the pr @moritzsternemann!

**Comment by stephengroat** [comment]
Please revert the `Added ProjectFi` commit. we like to do one site per PR

File: _data/utilities.yml [hunk 1]
```diff
@@ -143,6 +143,16 @@ websites:
       twitter: PowerStreamNews
       tfa: No
 
+    - name: Project Fi
```
Comment (stephengroat) [inline _data/utilities.yml:146]: please remove the projectfi

**Review by stephengroat — CHANGES_REQUESTED** [review]
Please revert the `Added ProjectFi` commit. we like to do one site per PR

**Comment by moritzsternemann** [comment]
Yep, sorry - I meant to create another branch :P

**Review by stephengroat — APPROVED** [review]

**Comment by stephengroat** [comment]
@moritzsternemann you have a merge conflict. could you resolve it?

File: _data/other.yml [hunk 2]
```diff
@@ -30,6 +30,13 @@ websites:
       sms: Yes
       software: Yes
 
+    - name: Any.do
+      url: https://web.any.do/
+      img: anydo.png
+      tfa: No
+      twitter: AnyDO
+      facebook: Any.DO
```
Comment (Jawshy) [inline _data/other.yml]: @`Any.do`\*

Comment (moritzsternemann) [reply _data/other.yml]: What do you mean?

Comment (Carlgo11) [reply _data/other.yml]: @moritzsternemann he means that you've incorrectly capitalized a couple of characters for the facebook name. It should be `Any.do` and not `Any.DO`

Comment (Jawshy) [reply _data/other.yml]: Yes. I should have clarified by initial comment. Anyway, thanks for correcting the case. `looks-good`

**Pull request merged**